Makefile

```diff
@@ -7,6 +7,7 @@ install:
 	@for vim in $(VIMS); do \
 		./scripts/install-vim $$vim; \
 	done
+	@./scripts/install-go-tools vim-8.0
```

Comment:
Same as the .travis.yml: if you try to install Vim 7.4 and you don't have vim 8.0, it will try to use a `vim` binary which doesn't exist.